src/swagger.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Well if you want to build the Swagger doc files

1) Run script to build this file
2) Run the swagger build script?

I mean this seems like a compiled asset to me, which doesn't need to be in version control... 

Compiled assets in Git = Horrible merge conflicts for no good reason.